boa3/analyser/typeanalyser.py

```diff
@@ -337,6 +401,78 @@ def visit_USub(self, sub: ast.USub) -> Operator:
         """
         return Operator.Minus
 
+    def visit_Eq(self, eq: ast.Eq) -> Operator:
```

Comment:
I think we should have a data structure to define the relationship between ast's operations and Operation's operations.
Something organized in a single location instead of declaring each of them on a different method, I know you need those methods separatelly, but you could delegate this resposability to a centralized code.

Reply:
Actually, I should ask: Do you need those methods separatelly? Are they being called internally by python tools?